site/docs/getting-started/connect-providers/azure-openai.md

```diff
@@ -9,6 +9,14 @@ sidebar_position: 3
 
 This guide will help you configure Envoy AI Gateway to work with Azure OpenAI's foundation models.
 
+## Supported Features
```

Comment:
let's do this "support matrix" in another PR for #705, so can you revert this for now.